forge-gui/res/cardsfolder/upcoming/frost_fair_lure_fish.txt

```diff
@@ -0,0 +1,12 @@
+Name:Frost Fair Lure Fish
+ManaCost:5 U R
+Types:Creature Fish
+PT:7/7
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create two 1/1 blue Fish creature tokens and two tapped Treasure tokens.
+SVar:TrigToken:DB$ Token | TokenAmount$ 2 | TokenScript$ u_1_1_fish,c_a_treasure_sac | TokenOwner$ You
```

Comment:
Tapped missing

Reply:
Won't that require engine work since all enter at the same time but only the treasures are tapped?

Reply:
need to use ChangeZoneResolve